src/main/scala/dpla/ingestion3/harvesters/QueryUrlBuilder.scala

```diff
@@ -34,8 +34,28 @@ class OaiQueryUrlBuilder extends QueryUrlBuilder with Serializable {
       .setParameter("verb", verb)
 
     params.get("resumptionToken") match {
```

Comment:
As much as I <3 pattern matches, I'm wondering if you could use `getOrElse(<default value>)` more in cases like this with Option or Map.